packages/type/src/errors.mts

```diff
@@ -0,0 +1,4 @@
+export class CancelPromptError extends Error {
```

Comment:
Not against doing this, but let's move all errors to keep them consistent if we go this route.